introduction.md

```diff
@@ -5,3 +5,9 @@ My name is k-snides, but you can call me k-snides.
 My fave path is all of them. I really like Code School. Learning lots on
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?